plugin.video.mlbbasesloaded/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.mlbbasesloaded" name="MLB BasesLoaded" version="0.2" provider-name="Jake Kramer">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.xbmcswift2" version="2.4.0"/>
+    <import addon="script.module.requests" version="2.9.1"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language></language>
+    <summary lang="en_GB">Redzone for MLB.tv</summary>
+    <description>Automatically shows you the best MLB game on. It will switch games at commercial breaks and will cut to a different game if it's significantly better than the current game.</description>
+  </extension>
+
+  <news>
+    v0.1  (2017-03-27)
+    - Initial version.
+
+    v0.2  (2017-04-04)
+    - Better synchronization between MLB.tv data and the stream.
+  </news>
+
```

Comment:
Also something I forgot to tell you, since krypton the folder layout is different. fanart.jpg and icon.png are both placed in the resources folder. You need to add to the addon.xml the `<assets>` tag defining both images. See here:
http://kodi.wiki/view/Add-on_structure
http://kodi.wiki/view/Addon.xml#.3Cassets.3E